src/Components/GameCard/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import star1 from './cardsIcons/etoile1.svg';
+import star2 from './cardsIcons/etoile2.svg';
+import star3 from './cardsIcons/etoile3.svg';
+import star4 from './cardsIcons/etoile4.svg';
+import star5 from './cardsIcons/etoile5.svg';
+import './index.css';
+
+const GameCard = ({ heroApi, cardSize }) => {
+  const [basicInfo, setBasicInfo] = useState([]);
+  const [stats, setStats] = useState([]);
+  const [visual, setVisual] = useState([]);
+  const [borderColor, setBorderColor] = useState('');
+  const [rarity, setRarity] = useState('');
+
+  useEffect(() => {
+    axios
+      .get(`https://www.superheroapi.com/api.php/10226516634046414/${heroApi}`)
+      .then((resp) => resp.data)
+      .then((data) => setBasicInfo(data));
+  }, []);
+
+  useEffect(() => {
+    axios
+      .get(`https://www.superheroapi.com/api.php/10226516634046414/${heroApi}`)
+      .then((resp) => resp.data)
+      .then((data) => setStats(data.powerstats));
+  }, []);
+
+  useEffect(() => {
+    axios
+      .get(`https://www.superheroapi.com/api.php/10226516634046414/${heroApi}`)
+      .then((resp) => resp.data)
+      .then((data) => setVisual(data.image));
+  }, []);
+
+  const rarityLevel = Math.round(
+    (parseInt(stats.power, 10) +
+      parseInt(stats.intelligence, 10) +
+      parseInt(stats.speed, 10) +
+      parseInt(stats.strength, 10) +
+      parseInt(stats.durability, 10) +
+      parseInt(stats.combat, 10)) /
+      6
+  );
+
+  useEffect(() => {
+    if (rarityLevel >= 80) {
+      setBorderColor('legendaryCard');
+      setRarity(star5);
+    } else if (rarityLevel >= 60 && rarityLevel < 80) {
```

Comment:
Replace condition with `if (rarityLevel >= 60` : you *already* know you're under 80 because of the previous test.
(Rince and repeat for next conditions)